src/Repositories/Body/IntBodyRepository.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Saloon\Repositories\Body;
+
+use Saloon\Exceptions\UnableToCastToStringException;
+use Saloon\Traits\Conditionable;
+use Saloon\Contracts\Body\BodyRepository;
+
+class IntBodyRepository implements BodyRepository
```

Comment:
Could we rename this to `IntegerBodyRepository`? Just personal preference :)

Reply:
sure!